### PR TITLE
Header & Footer

### PR DESCRIPTION
**Pull request by horsemaker**
- Added Header
- Added Footer

screenshot

screenshot

### CODE REVIEW

**Review by Chakravarthi0 — APPROVED** [review]
Clean code
good to merge

File: src/frontend/layouts/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export const Header = () => {
+  const { theme } = useTheme();
+
+  const navMenuForbiddenPaths = ["/", "signin", "signup"];
+  const { pathname } = useLocation();
+
+  return (
+    <header className="header">
+      <nav className="nav">
+        {!navMenuForbiddenPaths.includes(pathname) && (
```
Comment (msdeshmukh009) [inline src/frontend/layouts/Header/Header.js:17]: Nice way to remove unnecessary elements on the page.

**Review by msdeshmukh009 — APPROVED** [review]
The code & UI is looking really good. Good to merge!!

**Review by MiheerTamkhane — APPROVED** [review]
Code is Nice and easy to read and understandable. Good Work

**Pull request merged**